7a_temp_coop_munge/src/data_parsers/parse_vermillion_files.R

```diff
@@ -134,29 +170,44 @@ parse_Open_Water_Logger_2013 <- function(inind, outind) {
   raw <- data.table::fread(infile, skip = 1, col.names = c("num", "time",
                                                            "temp", "temp_avg",
                                                            paste0("V", 5:9)))
-  clean <- raw %>% mutate(DateTime = as.Date(time, format = "%m/%d/%y"),
-                          time = substr(time, 10,20),
-                          temp = fahrenheit_to_celsius(temp),
-                          depth = 8/3.28, DOW = '69037801') %>%
-    filter(time == "02:36:01 PM") %>% select(DateTime, temp, depth, DOW)
+  clean <- raw %>%
+    mutate(DateTime = as.Date(time, format = "%m/%d/%y"),
+           time = format(strptime(time, format = '%m/%d/%y %I:%M:%S %p', tz = 'Etc/GMT+6'), '%H:%M'),
+           timezone = 'GMT-6',
+           temp = fahrenheit_to_celsius(temp),
+           depth = 8/3.28,
```

Comment:
magic number - I'll stop commenting on these but there are others in rest of code